fastapi_profiler/profiler.py

```diff
@@ -109,3 +131,15 @@ async def get_profiler_result(self):
                     f.write(html_code)
 
             logger.info("Done writing profile to %r", html_file_name)
+        elif self._output_type == "prof":
+            prof_file_name = self.DEFAULT_PROF_FILENAME
+            if self._prof_file_name is not None:
+                prof_file_name = self._prof_file_name
+
+            logger.info(
+                "Compiling and dumping final profile to %r - this may take some time",
+                prof_file_name,
+            )
+
+            self._profiler.dump_stats(prof_file_name)
+            logger.info("Done writing profile to %r", prof_file_name)
```

Comment:
Output is pretty straightforward. I just dump the file to specified file name, I tried to follow the same conventions overall you used with HTML which put together is slightly duplicative but I think pretty readable